src/NetworkManagerUtils.c

```diff
@@ -112,28 +116,25 @@ get_new_connection_name (const GSList *existing_connections,
 		 * connection id. */
 		temp = g_strdup_printf (C_("connection id fallback", "%s %u"),
 		                        fallback_prefix, i);
-		for (iter = names; iter; iter = g_slist_next (iter)) {
-			if (nm_streq (iter->data, temp)) {
-				cname = temp;
-				goto found;
-			}
-		}
+
+		if (nm_utils_strv_find_first (NM_UNCONST_PPTR (char, existing_names),
```

Comment:
Could we use just: "(char **) existing names" instead of the macro? It seems more readable to me and I don't have to look up what the macro does.

Ok, this fixes the search.